packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -198,8 +198,8 @@ const DefaultSummaryConfiguration: ISummaryConfiguration = {
 
     maxTime: IdleDetectionTime * 12,
 
-    // Snapshot if 1000 ops received since last snapshot.
-    maxOps: 1000,
+    // Snapshot if 100 ops received since last snapshot.
+    maxOps: 100,
```

Comment:
I am curious why you chose 100? This means that we will now summarize 10 times more than before. Isn't that a big change?
Also, how do we measure this in order to tweak it? For example, if this results in very frequent summaries and slowing clients down, how will we know?

Reply:
Should this be behind a feature flag so that we can revert this if it causes regression?

Reply:
Vlad suggested 100 based on the telemetry data. The idea is that the consumers will still be able to override the values as they can today, via the config.

Reply:
Sounds good. My concern is that this may results in client slowing down. We should have a plan if that happens and maybe updating the container runtime options to provide this info is the answer.

Reply:
I'd prefer to split change into multiple PRs, with first one only getting rid of service-provided config by copying existing service config into client code. That way it will be so much easier to track what, when & why we changed over time

Reply:
ok. will revert all other changes and simply get rid of the service-provided config.